projects/canviewer/canviewer.py

```diff
@@ -0,0 +1,117 @@
+from canserver import init_can
+from gui.gui import Window
+
+import argparse
+import sys
+import time
+import signal
+import numpy as np
+import yaml
+import cantools
+
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import QTimer
+from rich.layout import Layout
+from rich.live import Live
+
+db = None
+
+with open("projects/canviewer/config.yml", "r") as config_file:
+    (
+        SHUTDOWN_NODES,
+        VEHICLE_VALUES,
+        VEHICLE_STATES,
+    ) = yaml.safe_load_all(config_file)
+# Messages that correspond to vehicle states/faults
+VEHICLE_STATE_SIGNALS = [
+    signal for message in VEHICLE_STATES.values() for signal in message.keys()
+]
+
+
+def sigint(signal_received, frame):
+    exit(0)
+
+
+def convertVtoT(x, Vin=3, R1=10000, R2=100000, T2=348.15, beta=3988):
```

Comment:
You should prob make a separate utils.py file that contains utilities that we use, including this function. It's not directly responsible for the CANviewer